src/com/Dryft/models/Driver.java

```diff
@@ -0,0 +1,25 @@
+package com.Dryft.models;
+
+public class Driver {
+    private final String name;
+    private final Car car;
+    private final Location location;
```

Comment:
Add driver gender and rating as the attributes

Reply:
This isn't done yet

Reply:
whats the datatype of rating?

Reply:
should i have two attributes, `rating` and `numReviews` and the method `updateRating` calculates the new rating and sets that

Reply:
Yeah, that seems good.Go ahead with this!